ferrocene/doc/user-manual/src/rustfmt/usage.rst

```diff
@@ -0,0 +1,30 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Using ``rustfmt``
+=================
+
+To format a single file, invoke ``rustfmt`` like this:
+
+.. code-block::
+
+   $ rustfmt file.rs
+
+You can also format multiple files at once like this:
+
+.. code-block::
+
+   $ rustfmt file1.rs file2.rs file3.rs
+
+Using ``rustfmt`` in safety critical setup
+------------------------------------------
+
+To use ``rustfmt`` in a safety critical setup you need to adhere to following workflow:
+
+1. Make a change to the source code of your crate.
+2. Commit or stage the change. The working tree must be clean.
+3. Run ``rustfmt`` on the Rust files of your crate.
+4. Perform following checks to ensure correct behaviour of ``rustfmt``
+    1. Compile your crate, to ensure that ``rustfmt`` did not introduce any syntax errors.
+    2. Review the changes ``rustfmt`` made, to ensure there are only syntactic, but no semantic changes.
+    3. Run your test suite, to ensure no semantic changes occured.
```

Comment:
This should not be part of the user manual, it should be in the "constraints" page of the safety manual for rustfmt.

Reply:
👍🏾